irodsManagerTest.py

```diff
@@ -616,5 +619,246 @@ def testParallelWrite(self):
             self.assertEqual(r['irods_response']['status_code'], 0)
 
 
+
+
+# Tests for resources operations
+class resourcesTests(unittest.TestCase):
+    def __init__(self, *args, **kwargs):
+        super(resourcesTests, self).__init__(*args, **kwargs)
+        self.api = irodsManager.manager(f'http://{HOSTNAME}:9001/irods-http-api/0.3.0')
+        self.adminToken = self.api.authenticate('rods', 'rods')
+        self.userToken1 = self.api.authenticate('jeb', 'ding')
```

Comment:
Is there a reason `__init__` is used rather than `setUp` (or `setUpClass`)?

Reply:
I set this up before I had looked at the tests from the API. I plan on changing it after finishing some of the more pressing issues.

Reply:
Cool cool.

Reply:
Opened issue #17